ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -176,6 +177,10 @@ public AccountDao(MySqlDataAccessor dataAccessor, MySqlAccountServiceConfig mySq
     listValidVersionSql =
         String.format("select %s, %s from %s " + "where (%s IS NULL or %s > now(3)) and %s = ? and %s = ? and %s = ?",
             VERSION, DELETE_TS, DATASET_VERSION_TABLE, DELETE_TS, DELETE_TS, ACCOUNT_ID, CONTAINER_ID, DATASET_NAME);
+    listVersionByModifiedTimeSql = String.format(
```

Comment:
there is a way in sql statement to skip first N rows before returning any data, could we just use that? I think LIMIT clause has two parameters to skip first N rows.

Reply:
Sure, updated it.